app/src/main/java/org/xbmc/kore/jsonrpc/method/AudioLibrary.java

```diff
@@ -85,7 +85,7 @@ public String resultFromJson(ObjectNode jsonObject) throws ApiException {
     /**
      * Retrieve all artists
      */
-    public static class GetArtists extends ApiListMethod<List<AudioType.DetailsArtist>> {
+    public static class GetArtists extends ApiMethod<ApiList<List<AudioType.DetailsArtist>>> {
```

Comment:
No, the goal is to have signature:
`ApiMethod<ApiList<AudioType.DetailsArtist>>`

`ApiList` substitutes `List` and encapsulates the `List` and `LimitsReturned`